### PR TITLE
[explorer] Use PageHeader component on pages, and add Banner component

### PR DESCRIPTION
**Pull request by Jordan-Mysten**
This updates pages to use the new `PageHeader` component everywhere. I had to update it to support `subtitle` based on the designs.

I also added the `Banner` component for the error display on transactions. I opted to not include this as part of the page header to keep the base component less opinionated.

Screen Shot 2022-10-19 at 12 23 18 PM
Screen Shot 2022-10-19 at 12 23 07 PM
Screen Shot 2022-10-19 at 12 22 43 PM
Screen Shot 2022-10-19 at 12 22 38 PM

### CODE REVIEW

**Comment by Jibz1** [comment]
LGTM. QQ does the banner scale to contain the message inside it?

**Review by Andrew47 — COMMENTED** [review]
This looks sufficient from an engineering perspective to unblock the Pay Transaction Work. However, we will need a review from @mystie711 to make sure this matches up with the Figma Design

**Review by Andrew47 — APPROVED** [review]
Great - we have buy-in from design

File: apps/explorer/src/pages/transaction-result/TransactionView.tsx
Comment (Jordan-Mysten) [reply apps/explorer/src/pages/transaction-result/TransactionView.tsx:395]: The banner stretches to the content of the message. And banners have a default icon so we should be good there.

**Pull request merged**
